storage/boltz/errors.go

```diff
@@ -76,3 +78,20 @@ func IsReferenceExistsError(err error) bool {
 func (err *ReferenceExistsError) Error() string {
 	return fmt.Sprintf("cannot delete %v with id %v is referenced by %v with id(s) %v, field %v", err.LocalType, err.LocalId, err.RemoteType, err.RemoteIds, err.RemoteField)
 }
+
+var testUniqueIndexDuplicateError = &UniqueIndexDuplicateError{}
```

Comment:
nit: `testUniqueIndexDuplicateError` seems a bit misnamed?

Reply:
I was duplicating the patterns in the file already.

Care to expand?